### DISCUSSION TITLE
Last call for testing / translations for Freeplane 1.10.5

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#801] [Announcements]
I am going to release Freeplane 1.10.5 soon. Please test the latest preview version 1.10.5_11 and submit your feedback and translations if needed. In the last preview, automatic screen resolution configuration replaces user configuration for Mac and Windows OS. Please let me know if it makes any trouble.

I have also removed some of the new API methods because of significant changes coming in layouts in upcoming Freeplane versions.

Regards,
Dimitry

**Comment by euu2021** [comment]
I don't know why, but I was using a screen size setting that was bigger than my actual monitor (I think that's because I randomly changed the preferences until I got the UI to the size that I liked). So, when I launched 1.10.5_11, the menu font size seemed bigger than before. 

The solution: I decreased the `Preferences…->Appearance->Look and feel->Menu item font size` and I got the same appearance that I'm used to.

**Reply by dpolivaev** [reply]
Yes, makes sense.

**Comment by dpolivaev** [comment]
Freeplane 1.10.5 is released
